dvc/command/experiments.py

```diff
@@ -489,21 +489,39 @@ def run(self):
             return 1
 
         if self.args.show_json:
-            import json
+            self._show_json(all_experiments)
+            return 0
+
+        metric_names, param_names = _collect_names(
+            all_experiments,
+            include_metrics=_parse_filter_list(self.args.include_metrics),
+            exclude_metrics=_parse_filter_list(self.args.exclude_metrics),
+            include_params=_parse_filter_list(self.args.include_params),
+            exclude_params=_parse_filter_list(self.args.exclude_params),
+        )
+
+        td = _experiments_td(
+            all_experiments,
+            metric_names,
+            param_names,
+            no_timestamp=self.args.no_timestamp,
+            sort_by=self.args.sort_by,
+            sort_order=self.args.sort_order,
+            precision=self.args.precision or DEFAULT_PRECISION,
+        )
 
-            ui.write(json.dumps(all_experiments, default=_format_json))
+        if self.args.show_csv:
+            self._show_csv(td)
         else:
-            show_experiments(
-                all_experiments,
-                include_metrics=self.args.include_metrics,
-                exclude_metrics=self.args.exclude_metrics,
-                include_params=self.args.include_params,
-                exclude_params=self.args.exclude_params,
-                no_timestamp=self.args.no_timestamp,
-                sort_by=self.args.sort_by,
-                sort_order=self.args.sort_order,
-                precision=self.args.precision or DEFAULT_PRECISION,
+            row_styles = lmap(baseline_styler, td.column("typ"))
+            styles = _get_styles(metric_names, param_names)
+            _post_process_td(td)
+            td.render(
```

Comment:
We could extend `render` to also print csv for us, WDYT?

Reply:
One problems `--show-json` still outside this render.

Reply:
`json` is completely different format from table/csv.